doc/tutorial/get_started.rst

```diff
@@ -225,11 +225,15 @@ Complete the following steps on each VM (``micro1``, ``micro2``, ``micro3``, and
       The ``--cohort="+"`` flag in the command ensures that the same version of the snap is installed on all machines.
       See :ref:`howto-snap-cluster` for more information.
 
-#. The LXD snap is already installed.
-   Refresh it to the latest version::
+#. If the LXD snap is already installed.
+   Refresh it to the latest LTS version::
```

Comment:
I am by far not good at this, but wouldn't the sentence be more fluid using a comma between `installed, refresh`?

Reply:
Whats with the double colons too?

Reply:
I just copied the double colons scheme which seems to be rendered as a single one: https://canonical-microcloud--294.com.readthedocs.build/en/294/tutorial/get_started/#install-microcloud-on-each-vm

Before checking the preview, I assumed the 2 lines were separated but that's not the case so I agree that `installed, refresh` reads better.

Reply:
How strange, is this normal @ru-fu ?

Reply:
Yes, that's RST syntax for introducing a code block. See https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/#code-blocks

Reply:
The double colons are apparently used before code blocks.